lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -3562,7 +3557,7 @@ namespace Js
         newFunctionBody->m_isFromNativeCodeModule = this->m_isFromNativeCodeModule;
     }
 
-    FunctionBody *
+    ParseableFunctionInfo *
```

Comment:
How about if we do that as a separate change?

-- Paul
________________________________
From: Yong Qu<mailto:notifications@github.com>
Sent: ‎1/‎30/‎2016 5:00 PM
To: Microsoft/ChakraCore<mailto:ChakraCore@noreply.github.com>
Cc: Paul Leathers<mailto:pleath@microsoft.com>
Subject: Re: [ChakraCore] Support for deferred parsing of HTML event handlers. (#234)


In lib/Runtime/Base/FunctionBody.cpp<https://github.com/Microsoft/ChakraCore/pull/234#discussion_r51353477>:

> @@ -3562,7 +3557,7 @@ namespace Js
>          newFunctionBody->m_isFromNativeCodeModule = this->m_isFromNativeCodeModule;
>      }
>
> -    FunctionBody *
> +    ParseableFunctionInfo *


It should be safe to get rid of the clone code now, replace with assert. I'll see if I have time to get rid of the caller side of the code.

—
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/ChakraCore/pull/234/files#r51353477>.